scala/scala.bzl

```diff
@@ -368,14 +368,23 @@ def _scala_macro_library_impl(ctx):
 # Common code shared by all scala binary implementations.
 def _scala_binary_common(ctx, cjars, rjars):
   write_manifest(ctx)
-  _compile_or_empty(ctx, cjars, [], False)  # no need to build an ijar for an executable
+  outputs = _compile_or_empty(ctx, cjars, [], False)  # no need to build an ijar for an executable
```

Comment:
should we give this a different name so its separate from the outputs defined on 379?